example/cluster-machine-template-multi-az.yaml

```diff
@@ -0,0 +1,242 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: cluster-api-multi-az
+  namespace: default
+  labels:
+    cni: "cluster-api-multi-az-crs-cni"
+    ccm: "cluster-api-multi-az-crs-ccm"
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: ["10.42.0.0/16"]
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: OscCluster
+    name:  cluster-api-multi-az
+    namespace: default
+  controlPlaneRef:
+    kind: KubeadmControlPlane
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+    name: "cluster-api-multi-az-control-plane"
+    namespace: default
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: OscCluster
+metadata:
+  name:  cluster-api-multi-az
+  namespace: default
+spec:
+  network:
+    clusterName: cluster-api-multi-az
+    loadBalancer:
+      loadbalancername: capo-k8s
+      clusterName: cluster-api-multi-az
+    net:
+      clusterName: cluster-api-multi-az
+    internetService:
+      clusterName: cluster-api-multi-az
+    natService:
+      clusterName: cluster-api-multi-az
+    bastion:
+      clusterName: cluster-api-multi-az
+      enable: false
+    subnets:
+    - name: cluster-api-multi-az-eu-west-2a
+      ipSubnetRange: ["10.42.0.0/24"]
```

Comment:
[ValidationError(OscCluster.spec.network.subnets[0].ipSubnetRange): invalid type for io.x-k8s.cluster.infrastructure.v1beta1.OscCluster.spec.network.subnets.ipSubnetRange: got "array", expected "string"
 
 It is string not array :)